crates/orchestrator/src/main.rs

```diff
@@ -24,4 +39,9 @@ async fn main() {
 
     tracing::info!("Listening on http://{}", address);
     axum::serve(listener, app).await.expect("Failed to start axum server");
+
+    if !OTEL_COLLECTOR_ENDPOINT.is_empty() {
```

Comment:
```suggestion
    let otel_enabled = !OTEL_COLLECTOR_ENDPOINT.is_empty()
```
and we can use this in both places

Reply:
Valid, implemented [here](https://github.com/madara-alliance/madara-orchestrator/pull/131/commits/ff2db549fab515b0dac23140e0a00a770eb24c04).